steps/reasons-for-appealing/evidence-upload/EvidenceUpload.js

```diff
@@ -0,0 +1,106 @@
+const { Question, goTo } = require('@hmcts/one-per-page');
+const { form, text } = require('@hmcts/one-per-page/forms');
+const { api } = require('../../../config/default');
+const { Logger } = require('@hmcts/nodejs-logging');
+const Joi = require('joi');
+const paths = require('paths');
+const formidable = require('formidable');
+const pt = require('path');
+const fs = require('fs');
+const request = require('request');
+
+class EvidenceUpload extends Question {
+  static get path() {
+    return paths.reasonsForAppealing.evidenceUpload;
+  }
+
+  static handleUpload(req, res, next) {
+    const pathToUploadFolder = './../../../uploads';
```

Comment:
Isn't it convention to use __dir when dealing with paths on Node/Express? For example:
`const pathToUploadFolder = '__dir/path/to/uploads'`

Reply:
Is it? Aren't you thinking to `__dirname` maybe?

Reply:
Doh, sorry yes, `__dirname`

Reply:
Anyway, I am resolving that later... but thanks to your comment I've discovered I had assigned that to a `const` beforehand, so I will remove that.